### PR TITLE
Added getting started guide

### PR DESCRIPTION
**Pull request by AmalieDue**
### Description
Added initial getting started guide.

### Relevant Issues/PRs
Closes DEV-136

### Type(s) of Change (Split the PR if you check many)
- [ ] Added user feature
- [ ] Added technical feature
- [ ] Added tests
- [ ] Refactor
- [ ] Bugfix
- [x] Updated documentation
- [ ] Kubernetes changes
- [ ] CI/CD workflow changes

### Testing Checklist
- [ ] I have added/updated unit tests for functional changes.
- [ ] I have added/updated integration tests for changes that affect dependent systems.
- [ ] I have added/updated end-to-end tests for feature changes.

### General Checklist
- [x] I have pulled in the latest changes from master.
- [ ] I have run `make lint`.
- [ ] I have successfully run `make tests`.
- [ ] I have updated relevant CI/CD workflows.
- [ ] I have updated relevant Kubernetes manifests/scripts.
- [x] I have updated/added relevant (internal and external) documentation (readme, doc comments, etc).
- [ ] I have updated the dependency map to reflect my changes.
- [ ] I have added the license to new source code files.
- [ ] I have spent some time looking over the full diff before creating this PR.

### CODE REVIEW

File: documentation/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+#### Step 2: Bootstrap an initial user
+
+```bash 
+docker exec <CONTAINER ID> ./encryptonize-core create-user <scopes>
+```
+
+Note down the generated credentials: `user_id` and `password`. They should be used to login to Encryptonize Core.
+
+#### Step 3: Login as initial user
```
Comment (philip-cc) [inline documentation/getting_started.md:35]: An initial paragraph about what we're about to do would be nice: "We can log in with the initial user by...."

File: documentation/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+#### Step 5: Use Encryptonize Core to encrypt an object
+In the following code example, the user will encrypt a very simple object.
+
+Input:
+```bash
+grpcurl -plaintext -H "authorization: bearer <access token>" 
+-d '{
+      "plaintext": "1234", 
+      "associated_data": "5678"
```
Comment (philip-cc) [inline documentation/getting_started.md:83]: Maybe something more illustrative like "sensitive plaintext data" and "metadata about the plaintext"

File: documentation/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+```
+Output:
+```bash
+{
+  "ciphertext": "Mpv3wlW0nHnBbXDtGT4Eqa24o57MpQpcxCiRj8M5SO1x56A=",
+  "associatedData": "5678",
+  "object_id": "a9a909b9-6019-488d-bfb4-b8b4243ae177"
+}
+```
+
```
Comment (philip-cc) [inline documentation/getting_started.md:95]: Briefly explain the output, e.g. the associated data is authenticated and bound to the ciphertext and the OID is needed to decrypt later.

File: documentation/getting_started.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+### Step 6: Shut down the Encryptonize Core instance
+
+You can either press `Ctrl-C`, or you can shut down to two running containers one at a time by running:
```
Comment (philip-cc) [inline documentation/getting_started.md:98]: This can be replaced with a simple `docker-compose down`.

File: documentation/getting_started.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+## Running Encryptonize Core locally (in Docker)
+In this section, you will be guided through how to get started with Encryptonize Core in Docker. This guide will use [gRPCurl](https://github.com/fullstorydev/grpcurl) to call the gRPC endpoints, but other tools can also be used.
+
+
+### Requirements
+- [Encryptonize Core](https://github.com/cyber-crypt-com/encryptonize-core)
+- [Docker](https://www.docker.com/get-started/)
+- [gRPCurl](https://github.com/fullstorydev/grpcurl) or another tool to call the gRPC endpoints.
+
+#### Step 1: Start a local Docker Compose instance
```
Comment (philip-cc) [inline documentation/getting_started.md:12]: There should probably be some mention of the config file with a reference to the manual.

File: documentation/getting_started.md (hunk 2 above)
Comment (AmalieDue) [reply documentation/getting_started.md:83]: It gives me an "Illegal base64 data" error. Should I convert the strings in some way?

Comment (philip-cc) [reply documentation/getting_started.md:83]: Ah, right, I forgot that it expects base64. We can leave it like this then.

File: documentation/getting_started.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
+# Getting started
+
+## Running Encryptonize Core locally (in Docker)
+In this section, you will be guided through how to get started with Encryptonize Core in Docker. This guide will use [gRPCurl](https://github.com/fullstorydev/grpcurl) to call the gRPC endpoints, but other tools can also be used.
```
[the remark this answers is not in the document]
Comment (AmalieDue) [reply documentation/getting_started.md:5]: Fixed

File: documentation/getting_started.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+
+```bash
+docker ps -a
+```
+The output will show that two containers, `encryptonize-core` and `cockroachdb`, are running.
+
+#### Step 2: Bootstrap an initial user
+
+```bash 
+docker exec <CONTAINER ID> ./encryptonize-core create-user <scopes>
```
[the remark this answers is not in the document]
Comment (AmalieDue) [reply documentation/getting_started.md]: Fixed

File: documentation/getting_started.md (hunk 1 above)
Comment (AmalieDue) [reply documentation/getting_started.md:39]: Fixed

File: documentation/getting_started.md (hunk 3 above)
Comment (AmalieDue) [reply documentation/getting_started.md:101]: Fixed

File: documentation/getting_started.md (hunk 4 above)
Comment (AmalieDue) [reply documentation/getting_started.md]: Fixed

File: documentation/getting_started.md (hunk 5 above)
Comment (AmalieDue) [reply documentation/getting_started.md:14]: Fixed

File: documentation/getting_started.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+```
+the output will show that two containers, `encryptonize-core` and `cockroachdb`, are running.
+
+#### Step 2: Bootstrap an initial user
+
+```bash 
+docker exec encryptonize-core ./encryptonize-core create-user m
+```
+
+The `m` at the end of the code line above means that the user is provided with the `USERMANAGEMENT` scope. A user can be provided with any number of scopes, see a list of all possible scopes in the [Managing Users](user_manual.md#managing-users) section in the user manual. If a user should be provided with all possible scopes, then the `m` above should be replaced by `rciom`. Note that a user must be provided with the `USERMANAGAMENT` scope in order to be able to add other users which is why `m` was given in the code example above.
```
Comment (philip-cc) [inline documentation/getting_started.md:35]:
```suggestion
The last parameter sets the scopes for the initial user. For more details on scopes see the the [Managing Users](user_manual.md#managing-users) section in the user manual. In this example we create the user with the `USERMANAGEMENT` scope. Note that a user must be provided with the `USERMANAGAMENT` scope in order to be able to create other users.
```

**Review by philip-cc — APPROVED** [review]

**Pull request merged**